src/nyc_trees/apps/survey/fixtures/species.json

```diff
@@ -108,12 +108,12 @@
         "model": "survey.Species",
         "fields": {
             "species_code": "ACPLCR",
-            "scientific_name": "Acer platanoides",
+            "scientific_name": "Acer platanoides 'Crimson King'",
             "created_at": "2023-04-10T17:11:50.350Z",
             "updated_at": "2023-04-10T17:11:50.350Z",
             "forms_id": 92393,
-            "common_name": "Crimson King Norway Maple",
-            "cultivar": "Not Green Leaf"
+            "common_name": "Crimson King Maple",
+            "cultivar": ""
```

Comment:
The row in ``Species_List_rev3.xlsx`` with forms_id 92393 still has a cultivar value of "Not Green Leaf". Are we dropping it here for a specific reason?